responsemanager/runtraversal/runtraversal.go

```diff
@@ -12,6 +12,15 @@ import (
 
 var logger = logging.Logger("gs-traversal")
 
+type errorString string
+
+func (e errorString) Error() string {
+	return string(e)
+}
+
+// ErrFirstBlockLoad indicates the traversal was unable to load the very first block in the traversal
+const ErrFirstBlockLoad = errorString("Unable to load first block")
```

Comment:
Why not just use errors.New()?

Reply:
cause then it's a VAR rather than a const. I've always been anxious about golang errs being vars. cause theoretically anyone can overwrite them, so I often employ this pattern. See https://dave.cheney.net/tag/errors